### PR TITLE
CMake, vcpkg manifest and github actions

### PR DESCRIPTION
**Pull request by lstipakov**

### CODE REVIEW

**Comment by lstipakov** [comment]
You can see the effect of github actions here: https://github.com/lstipakov/openvpn-gui/actions/runs/838850412

**Review by chipitsine — APPROVED** [review]

File: main.c [hunk 1]
```diff
@@ -177,7 +177,7 @@ int WINAPI _tWinMain (HINSTANCE hThisInstance,
   mgmt_rtmsg_handler handler[] = {
       { ready,    OnReady },
       { hold,     OnHold },
-      { log,      OnLogLine },
+      { log_,      OnLogLine },
```
Comment (selvanair) [inline main.c:180]: While at it shall we decorate all those enums with an underscore -- like echo_ and ready_? Or all caps?

Comment (lstipakov) [reply main.c:180]: Agreed, I'll add _ to the rest enum names for consistency.

Comment (selvanair) [reply main.c:180]: Looks good now. 
For other changes, you now better and @chipitsine has approved. Merging.

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    proxy.c
+    registry.c
+    save_pass.c
+    scripts.c
+    service.c
+    tray.c
+    viewlog.c
+    res/openvpn-gui-res.rc)
+
+configure_file(config-msvc.h config.h COPYONLY)
```
Comment (selvanair) [inline CMakeLists.txt:28]: @lstipakov  Currently PACKAGE_VERSION_RESOURCE is hard coded in config-msvc.h. For MSVC build how to extract it from configure.ac during build time?

Comment (lstipakov) [reply CMakeLists.txt:28]: Good point. That's how: https://github.com/OpenVPN/openvpn-gui/pull/427

**Pull request merged**